File: examples/H20/robots/sdk_package/example/h2x/high_level/squat_up.cpp
#include <iostream>

#include "h2x/h2x_high_level.h"

using namespace humanoid;
using namespace humanoid::h2x;

int main() {
  std::cout << "=== h2x 蹲起示例 ===" << std::endl;
  H2xHighLevel high_level;

  int ret = high_level.SquatUp();
  if (ret == 0) {
    std::cout << "成功蹲起" << std::endl;
  } else {
    std::cout << "蹲起失败, 错误码：" << ret << std::endl;
    return -1;
  }
  return 0;
}

File: examples/H20/robots/sdk_package/example_py/h2x/high_level/squat_up.py
import humanoid_sdk_py.h2x as h2x
import time

h2x_high_level = h2x.H2xHighLevel()
time.sleep(1)

print("=== h2x 蹲起示例 ===")
ret = h2x_high_level.SquatUp()
if ret == 0:
    print("蹲起成功")
else:
    print("蹲起失败，错误码：", ret)
